.github/workflows/ci-pullrequest-main.yml

```
@@ -151,4 +151,10 @@ jobs:
       uses: docker/setup-buildx-action@v2
     
     - name: Docker Build
-      run: docker image build src/Dmr.Api/
+      run: docker image build src/Dmr.Api/ -t dmr
```

Please make this consistent with the other repositories:

```suggestion
      run: docker image build -f src/Dmr.Api/Dockerfile ./src -t dmr
```